functions/gdl/game_controller.py

```diff
@@ -38,11 +46,24 @@
 @API.route('/', strict_slashes=False)
 class Games(Resource):
 
-    @API.doc('List of available games', params={'language': 'Optional BCP47 language code to filter results'})
-    @API.marshal_list_with(game, skip_none=True)
+    @API.doc('List of available games', params={'language': 'Optional BCP47 language code to filter results', 'page': 'Return results for this page', 'page-size': 'Return this many results per page.' })
+    @API.marshal_with(game_list, skip_none=True)
     def get(self):
         lang = flask.request.args.get('language')
-        return game_repository.all_with_language(lang) if lang else game_repository.all()
+        page = flask.request.args.get('page')
+        page_size = flask.request.args.get('page-size')
+        if page:
+            try:
+                page = int(page)
+            except ValueError:
+                API.abort(400, "Invalid page integer: {}".format(page))
+        if page_size:
+            try:
+                page_size = int(page_size)
+            except ValueError:
+                API.abort(400, "Invalid page_size integer: {}".format(page_size))
+
+        return game_repository.all_with_language(lang) if lang else game_repository.all(page, page_size)
```

Comment:
Her ser det ut til at du dropper paging, dersom man spesifiserer `lang`. Så om jeg vil få listet opp alle spill på engelsk, så får jeg alle, men uten paging. Det er vel ikke helt heldig?

Reply:
I bok-apiet så har vi en `v1/books/{lang}` i tillegg til `v1/books`. Jeg ser ikke behovet for å separere det slikt, så jeg gjort det slikt at default språket mann spørr etter er engelsk og `lang` parameteret er optional.